servicetalk-grpc-gradle-plugin/plugin-config.gradle

```diff
@@ -27,7 +27,7 @@ dependencies {
 gradlePlugin {
   plugins {
     serviceTalkGrpcPlugin {
-      id = "servicetalk-grpc"
+      id = "io.servicetalk.servicetalk-gradle-plugin-grpc"
```

Comment:
I would propose `io.servicetalk.servicetalk-grpc-gradle-plugin` id here. It will generate files for Gradle Plugin Portal in the same folder where we have a maven artifact, in a subfolder with the name: `io.servicetalk.servicetalk-grpc-gradle-plugin.gradle.plugin`.

Reply:
Ok let me try and see what it looks like